### PR TITLE
Universal/docs lighting

### PR DESCRIPTION
**Pull request by reallyfancy**
---
### Purpose of this PR
Adding Lighting section to URP docs, including Light component page so that we can link to it from main Unity Manual.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/lighting.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Using the Universal Render Pipeline (URP), you can achieve realistic lighting that is suitable for a range of art styles.
+
+All of Unity's render pipelines share common lighting functionality, but each render pipeline has some important differences.
+
+Areas where the Universal Render Pipeline (URP) differs from Unity's common lighting functionality are:
+
+* The [Light component inspector](light-component.md), which displays some URP-specific controls.
+* The [Universal Additional Light Data](universal-additional-light-data.md) component, which allows Unity to store Light-related data that is specific to URP.
+* Enlighten is not supported in URP, either for lightmapping or for Realtime Global Illumination. For more information, see this [Unity blog post](https://blogs.unity3d.com/2019/07/03/enlighten-will-be-replaced-with-a-robust-solution-for-baked-and-real-time-giobal-illumination/?_ga=2.246542978.783393071.1580122240-359214009.1520341967).
```
Comment (Verasl) [inline com.unity.render-pipelines.universal/Documentation~/lighting.md]: Dunno if we want to change this or not, but baking static linghtmaps with enlighten does work in URP, it's just the realtime part that is not supported in 19.3. But it is getting ripped out of 2020 I believe so won't even be a selectable option?

File: com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md [hunk 2]
```diff
@@ -19,6 +19,15 @@ This table provides an overview of the current features supported in the Univers
...
+| ***Batching***                                               |                                                              |                                                              |
+| Static Batching (By Shader)                                  | Not supported                                                | Yes                                                          |
+| Static Batching (By Material)                                | Yes                                                          | Yes                                                          |
+| Dynamic Batching                                             | Yes                                                          | Yes                                                          |
+| Dynamic Batching (Shadows)                                   | Yes                                                          | In research                                                  |
+| GPU Instancing                                               | Yes                                                          | Yes                                                          |
```
Comment (Verasl) [inline com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md]: do we want to mention this is disabled when using the SRP Batcher? as I have seen people wondering why it isn't working by default?

Comment (reallyfancy) [reply com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md]: Yes, but not in this page or PR :)

File: com.unity.render-pipelines.universal/Documentation~/lighting.md (hunk 1 above)
Comment (reallyfancy) [reply com.unity.render-pipelines.universal/Documentation~/lighting.md]: Updated!

**Pull request merged**